### DISCUSSION TITLE
Insert multiple entities in each room without using groups

### DISCUSSION THREAD

**Discussion by Giuseppe-P** [dwainscheeren/dwains-lovelace-dashboard#283] [General]
how can i insert multiple entities in each room without using groups?
example: sew - light 1 - light 2 etc.

**Comment by AalianKhan** [comment]
From what I am understanding, on the room page, you want 2 buttons for your sew light 1 and sew light 2 instead of opening a light page to see all of your lights. 

You can do this by using page entities (https://dwainscheeren.github.io/dwains-lovelace-dashboard/configuration/rooms.html#page_entities-object)
Or if that doesn't fit your needs then you can use room addon's to add a custom button to your page (https://github.com/dwainscheeren/dwains-lovelace-dashboard/discussions/267)

**Reply by Giuseppe-P** [reply]
Exactly you understood my need. I'll try your directions. Thanks again.

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
